platform/viewer/src/connectedComponents/ToolbarRow.js

```diff
@@ -150,46 +150,88 @@ class ToolbarRow extends Component {
   }
 }
 
+function _getCustomButtonComponent(button, activeButtons) {
+  const CustomComponent = button.CustomComponent;
+
+  function isValidComponent(Component) {
+    return (typeof Component === 'function');
+  }
```

Comment:
I don't understand the advantage of creating a temporary function instead of simple variable that explains the result/check?

Reply:
The unique advantage is for maintainability purpose.. but in other hand it will cost a bit more of memory usage.. Changed it to variable declaration instead